api/src/test/scala/dcos/metronome/api/v1/controllers/JobSpecControllerTest.scala

```diff
@@ -1,10 +1,10 @@
 package dcos.metronome.api.v1.controllers
 
 import dcos.metronome.api.v1.models._
-import dcos.metronome.api.{ MockApiComponents, OneAppPerTestWithComponents, TestAuthFixture, UnknownJob }
+import dcos.metronome.api._
```

Comment:
Wildcard import? 😱 
Well in test code it might be OK – IIRC our conventions are: No?

Reply:
:) fixed